exercises/practice/sublist/.approaches/introduction.md

```diff
@@ -0,0 +1,58 @@
+# Introduction
+There are two broad ways to solve Sublist. 
+
+## General guidance
+To write the code, you need to branch out (probably with `if`) into the four different possible conditions, and return the appropriate name of the category. 
+
+## Approach: list manipulation
+The direct approach would be to manipulate and check the given lists to solve this.
+This solution uses a helper function, which simplifies things, but the approach can be implemented without it.
+
+```python
+SUBLIST = 1
+SUPERLIST = 2
+EQUAL = 3
+UNEQUAL = 4
+
+def check_sub_sequences(list_one, list_two):
+    n1 = len(list_one)
+    n2 = len(list_two)
+    return any(list_two[i:i+n1] == list_one for i in range(n2 - n1 + 1))
+    
+def sublist(list_one, list_two):
+    if list_one == list_two:
+        return EQUAL
+    if check_sub_sequences(list_one, list_two):
+        return SUBLIST
+    if check_sub_sequences(list_two, list_one):
+        return SUPERLIST
+    return UNEQUAL
+```
+
+Read more on the [detail of this approach][approach-list-manipulation].
+
+## Approach: using strings
+Another clever approach is to convert the lists to strings and then use the `in` operator to check for sub-sequences.
```

Comment:
This approach is demonstratively broken. A test was explicitly added which breaks the simple non `+ ","` solution to illustrate that this approach is incorrect. Adding the `+ ","` circumvents that specific test but doesn't change the fact that this approach is broken and will give incorrect results for well crafted inputs.

This can be an interesting opportunity to _show_ why it is simple and tempting to use, but is, in fact, broken. However, it should not be listed as a valid solution.

Reply:
```python
>>> l1 = ["1", "2"]
>>> l2 = ["5", "'1', '2',", "7"]
>>> sublist(l1, l2) == SUBLIST
True
```

If you're converting a list to a string representation, that string representation can be placed into another list as a single element. The [latest commit to the canonical data for that exercise](https://github.com/exercism/problem-specifications/pull/1866) attempts to capture that ... for the simplest string representation.

Students can add any arbitrary string into the representation to try to "defeat" this test, eg (roughly) `l1_check = str(list_one) + TOKEN`. It's impossible to add a static test which covers every TOKEN value. However, we did add a test for `TOKEN = ""`.

Reply:
@IsaacG thanks for the clarification!

How does this look?